src/examples/USEnrichmentSecondaryExample.cs

```diff
@@ -0,0 +1,116 @@
+namespace Examples
+{
+	using System;
+	using System.Net;
+	using System.Collections.Generic;
+	using System.IO;
+    using System.Linq;
+    using SmartyStreets;
+	using SmartyStreets.USEnrichmentApi;
+    using System.Reflection;
+    using System.Text;
+
+	internal static class USEnrichmentSecondaryExample
+	{
+		public static void Run()
+		{
+            // specifies the TLS protocoll to use - this is TLS 1.2
+            const SecurityProtocolType tlsProtocol1_2 = (SecurityProtocolType)3072;
+
+            // var authId = "Your SmartyStreets Auth ID here";
+            // var authToken = "Your SmartyStreets Auth Token here";
+
+            // We recommend storing your keys in environment variables instead---it's safer!
+            var authId = Environment.GetEnvironmentVariable("SMARTY_AUTH_ID");
+			var authToken = Environment.GetEnvironmentVariable("SMARTY_AUTH_TOKEN");
+			ServicePointManager.SecurityProtocol = tlsProtocol1_2;
+
+			var client = new ClientBuilder(authId, authToken).BuildUsEnrichmentApiClient();
+			
+            // See the US Enrichment API documenation for all available datasets and data subsets https://www.smarty.com/docs/cloud/us-address-enrichment-api#data-sets
+            SmartyStreets.USEnrichmentApi.Secondary.Result[] results = null;
+            var lookup = new SmartyStreets.USEnrichmentApi.Secondary.Lookup("325023201");
+            // Options available for Lookup
+            // lookup.SetEtag("HAYDKMJXHA4DKNA");
+
+            try {
+                results = client.SendSecondaryLookup(lookup);
+            }
+            catch (NotModifiedException ex) {
+                Console.WriteLine(ex.Message); // The Etag value provided represents the latest version of the requested record
+            }
+            catch (Exception ex) {
+                Console.WriteLine(ex.Message + ex.StackTrace);
+            }
+            
+            if (results != null) {
```

Comment:
Depending on the version of C#, always prefer "results is not null" to "results != null" because the != operator can be overloaded and it is considered safer.

Reply:
Let's consider this for a future change to the sdk.